lang/en.json

```diff
@@ -177,6 +177,8 @@
         "ExceptionDatabaseVersion": "Your %1$s version is %2$s but Matomo requires at least %3$s.",
         "ExceptionDatabaseVersionNewerThanCodebase": "Your Matomo codebase is running the old version %1$s and we have detected that your Matomo Database has already been upgraded to the newer version %2$s.",
         "ExceptionDatabaseVersionNewerThanCodebaseWait": "Maybe your Matomo administrators are currently finishing the upgrade process. Please try again in a few minutes.",
+        "ExceptionDatabaseUnavailable": "Database server has gone away",
```

Comment:
@justinvelluppillai be great to still use `MySQL server has gone away` so when people search for a solution they will find our FAQ (it's the most viewed FAQ we have AFAIK). Also it helps people find other more relevant content when they search for the error.